generators/server/templates/src/main/resources/config/_application-prod.yml

```diff
@@ -181,6 +181,9 @@ jhipster:
             host: localhost
             port: 2003
             prefix: <%= baseName %>
+        logs: # report metrics in the logs
+            enabled: false
+            reportFrequency: 60 # in seconds
```

Comment:
Actually I don't really know what default frequency should be used. The spark reporter also report logs every minute and does not even provide a way to configure the frequency in the app properties.
Anyway, It should not have too much performance impact as it is only sending a few dozens of logs every minute minute.